scheme/common/arm/handlers.go

```diff
@@ -182,8 +182,17 @@ func MatchPlatformConfig(scheme string, evidence psatoken.IClaims, endorsements
 	if err != nil {
 		return false
 	}
-	if len(endorsements) > 1 {
-		log.Errorf("got %d CCA configuration endorsements, want 1", len(endorsements))
+
+	endorsementsLen := len(endorsements)
+
+	switch endorsementsLen {
+	case 0:
```

Comment:
This suggestion is made under the yet-to-be-decided assumption that config ID is mandatory.

Can we move that (design) discussion to a separate issue (as suggested in https://github.com/veraison/services/pull/264#issuecomment-2315011210) and move on?

If we decide that config ID is mandatory, this switch statement will be amended accordingly in the scope of that activity.

Reply:
I am happy to raise the issue and will raise it in the CoRIM repository and also in CCA Endorsement Repository to clarify in either direction (whether Platform Config is Mandatory or Optional). Till that point where we have a conclusion, can we keep the implementation consistent and not treat Config Reference Value any different from SW Measurement of the Components. 
In the current logic of PSA and CCA if no RV for SW Components is located, we fail the attestation, so the same should be the treatment with the Platform Config, why treat it differently ? Then if post design discussion we conclude that the optionality is to be preserved, we should introduce the `case 0`. I am still not sure, where this test case is coming from where you are testing, can you please also provide some context of the test..

Reply:
> In the current logic of PSA and CCA if no RV for SW Components is located, we fail the attestation, so the same should be the treatment with the Platform Config, why treat it differently ? 

Maybe their failure to provide it indicates they don't particularly care?

It's unclear to me whether config ID is always conveying something relevant appraisal-wise.  I haven't fully grasped the precise semantics of config ID, and therefore its implication on the overall TCB appraisal is not yet obvious to me.

Reply:
> I am still not sure, where this test case is coming from where you are testing, can you please also provide some context of the test..

We are testing against the Linaro public endpoint, which has been provisioned without a config ID reference value since the platform that issues the token is emulated and the config ID is just set to a random value.

Reply:
Thank you! I have raised https://github.com/yogeshbdeshpande/draft-cca-rats-endorsements/issues/2
to track this. Please feel free to leave the PR Open and we can discuss the above issue and then conclude the discussion here!

Reply:
> Thank you! I have raised [yogeshbdeshpande/draft-cca-rats-endorsements#2](https://github.com/yogeshbdeshpande/draft-cca-rats-endorsements/issues/2) to track this.

Great.

> Please feel free to leave the PR Open and we can discuss the above issue and then conclude the discussion here!

Instead, I think we should merge this and revisit the code when we have a stable design.